### DISCUSSION TITLE
Is high system RAM usage normal?

### DISCUSSION THREAD

**Discussion by frenchiveruti** [vladmandic/sdnext#2406] [Q&A]
Hi, I tried following this guide: 
https://github.com/vladmandic/automatic/wiki/Installation 

Which has many different things outdated, assumes a lot of things about the knowledge on the end-user and well, quite frankly I don't know if I did a correct installation. 

I have a RTX3060 Ti(8GB), 32GB RAM, i7-4790K. 

I'm noticing that the website alone, while idle, after following the guide, uses ~20GB of RAM. Is that normal? 
![image](https://github.com/vladmandic/automatic/assets/1095997/fd1c7906-adc0-4070-b59b-266a92f53236)

**Comment by Tillerz** [comment]
There's 4 Powershells, did you check what memory they eat each, maybe some old stuck SD process or a second copy your started?

**Comment by vladmandic** [comment]
any model is loaded into ram and then moved in and out of vram as needed. and size of model as well as compute settings such as fp16 vs fp32 impact how much memory is used. and none of those are visible here, so i cannot say more.

**Reply by frenchiveruti** [reply]
Which one would consume the least amount of VRAM, apparently the model is not meant to run on 8GB of VRAM, right?
